crossbeam-utils/src/atomic/mod.rs

```diff
@@ -1,5 +1,16 @@
 //! Atomic types.
 
+cfg_if! {
+    // TODO(jeehoonkang): want to express `target_pointer_width >= "64"`, but it's not expressible
```

Comment:
In principle Rust also supports [a 16-bit target](https://github.com/rust-lang/rust/blob/9ebf47851a357faa4cd97f4b1dc7835f6376e639/src/librustc_target/spec/msp430_none_elf.rs) for which two `AtomicUsize`s would make a 32-bit counter. I think this can safely be ignored as it's very primitive hardware, but mentioning it here for completeness. There are no 128-bit targets currently as far as I can tell from a quick search.

Reply:
I added some documentation: https://github.com/crossbeam-rs/crossbeam/pull/359/files#diff-7e838416d7e6b94eeac80063d0b6a446R10